src/mlpack/tests/main_tests/adaboost_test.cpp

```diff
@@ -41,25 +41,24 @@ struct AdaBoostTestFixture
   }
 };
 
-BOOST_FIXTURE_TEST_SUITE(AdaBoostMainTest, AdaBoostTestFixture);
-
 /**
  * Check that number of output labels and number of input
  * points are equal.
  */
-BOOST_AUTO_TEST_CASE(AdaBoostOutputDimensionTest)
+TEST_CASE_METHOD(AdaBoostTestFixture, "AdaBoostOutputDimensionTest",
+                 "AdaBoostMainTest")
```

Comment:
```suggestion
                 "[AdaBoostMainTest][BindingTests]")
```
We should probably do this for the other tests in this file too; what do you think?